src/framegrab/cli/autodiscover.py

```diff
@@ -21,17 +21,17 @@
     show_default=True,
 )
 @click.option(
-    "--rtsp_discover_modes",
+    "--rtsp_discover_mode",
     type=click.Choice(PREVIEW_RTSP_COMMAND_CHOICES, case_sensitive=False),
-    default="light",
+    default="disable",
```

Comment:
Nit, but 'disable' is a weird choice for the default. 'disable' implies the default should be to search for rtsp urls and we're going out of our way to turn it off. 'off' or 'none' would make more sense to me

Reply:
Yea I agree, `off` would be a better naming for this.